zulipterminal/ui_tools/boxes.py

```diff
@@ -80,6 +86,28 @@ def stream_box_view(self, button: Any=None, caption: str='',
         ]
         self.contents = write_box
 
+    def autocomplete(self, text: Any, state: int) -> Any:
+        if text.startswith('@'):
+            users = self.view.users.copy()
+            users_list = users.copy()
+            for user in users:
+                if not match_user(user, text[1:]):
+                    users_list.remove(user)
+            tmp = []
+            if text:
```

Comment:
Can this condition ever be `False`?

Reply:
this condition will remove all the not matching names from the list of users